scripts/log-capture

```diff
@@ -29,20 +29,20 @@ _to_log dmesg
 _to_log dmidecode
 _to_log lspci -vvnn
 
-for DEV_NAME in $(list-harddrives | cut -d " " -f 1)
+for DEV_NAME in $(cd /sys/block ; ls);
```

Comment:
Nicer done without cd, for DEV_NAME in $(ls /sys/block/);

Reply:
Fixed